src/pages/index.js

```diff
@@ -135,30 +224,107 @@ function closeOverlay(event) {
   }
 }
 
-modals.forEach((modal) => {
-  modal.addEventListener("mousedown", closeOverlay);
-});
+// modals.forEach((modal) => {
+//   modal.addEventListener("mousedown", closeOverlay);
+// });
 
 function handleEditFormSubmit(event) {
   event.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
 
-  closeModal(editModal);
+  //change text content to "Saving..."
+  const submitButton = event.submitter;
+  submitButton.textContent = "Saving...";
```

Comment:
`submitButton.textContent = "Saving...";`


It's not needed here because it's done inside `setButtonText(submitButton, true);` in line 237

So, you'll have only this:

```js
  //change text content to "Saving..."
  const submitButton = event.submitter;
  setButtonText(submitButton, true);

```